std/io.zig

```diff
@@ -461,7 +461,7 @@ pub const SliceOutStream = struct {
 
     pub stream: Stream,
 
-    pos: usize,
+    pub pos: usize,
```

Comment:
does it mean that ``pos`` field can be changed from outside of this class? 
- I think it is an implementation detail and shouldn't be open
- Changes on this field from outside of this class can cause problems later
- It was made ``pub`` because of a unit test, which is a code smell
- it is not coherent, since ``SliceInStream.pos`` is not ``pub``

Simply just use the ``getWritten`` method.